### PR TITLE
[risk=low][no ticket] Remove all npmrc / Font Awesome logic from UI builds

### PR DESCRIPTION
**Pull request by jmthibault79**
I was able to build the UI locally after a full clean (including rm node_modules).  Running Local UI+API did not have any obvious problems.

---
**PR checklist**

- [x] I have included an issue ID or "no ticket" in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [x] I have included a risk tag of the form `[risk=no|low|moderate|severe]` in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [x] I have manually tested this change and my testing process is described above.
- [ ] This change includes appropriate automated tests, and I have documented any behavior that cannot be tested with code.
- [x] I have added explanatory comments where the logic is not obvious.
- One or more of the following is true:
  - [ ] This change is intended to complete a JIRA story, so I have checked that all AC are met for that story.
  - [ ] This change fixes a bug, so I have ensured the steps to reproduce are in the Jira ticket or provided above.
  - [ ] This change impacts deployment safety (e.g. removing/altering APIs which are in use), so I have documented the impacts in the description.
  - [ ] This change includes a new feature flag, so I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later.
  - [ ] This change modifies the UI, so I have taken screenshots or recordings of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P).
  - [ ] This change modifies API behavior, so I have run the relevant E2E tests locally because API changes are not covered by our PR checks.
  - [x] None of the above apply to this change.

### CODE REVIEW

**Comment by jmthibault79** [comment]
I'm seeing a different result when I'm logged into CircleCI vs. locally.

```
[local]
$ yarn install
[completes normally]
```

```
[circle]
$ yarn install                                                                                                                                                                       
yarn install v1.22.19
[1/4] Resolving packages...                                                                                                                                                                                              [2/4] Fetching packages...
error An unexpected error occurred: "https://npm.fontawesome.com/@fortawesome/fontawesome-common-types/-/6.4.2/fontawesome-common-types-6.4.2.tgz: Request failed \"401 Unauthorized\"".
info If you think this is a bug, please open a bug report with the information provided in "/home/circleci/workbench/ui/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

File: ui/.npmrc [hunk 1]
```diff
@@ -1,3 +1,3 @@
-@fortawesome:registry=https://npm.fontawesome.com/
+@fortawesome:registry=https://registry.yarnpkg.com/
```
Comment (jmthibault79) [inline ui/.npmrc:1]: We're seeing "401 Unauthorized" problems on CircleCI when it tries to access the fontawesome repo to download even the free packages.

Force the yarn registry instead, so hopefully we won't attempt to access fontawesome at all.

Comment (jmthibault79) [reply ui/.npmrc:1]: We may be able to remove this at some point in the future, after whatever it is happening in Circle (caches?) adjusts

**Review by freemabd — APPROVED** [review]
LGTM

**Pull request merged**